packages/tools/src/types/AnnotationTypes.ts

```diff
@@ -7,6 +7,8 @@ type Annotation = {
   highlighted?: boolean;
   /** If the annotation is locked for manipulation */
   isLocked?: boolean;
+  /** If the annotation is visible for manipulation */
+  isVisible?: boolean;
```

Comment:
Why this is optional? I thought it is always there and default to true

Reply:
This is indeed a very trick one. It tells us something about our architecture: 
As far as OOO is concerned, this type is not what it meant to be. 
When I see this type Annotation, I understand this represents an instance of an AnnotationTool. If that is true, each annotation instance has at least to 2 phases/shapes types. I.e at construction time and after adding the handles.
So, with all that said that is why many of flags here are optional, because they are completed (if that happens) at run time. 
In summary, AnnotationType is being used as a representation of instance (at construction time) and considering this flag is added by decoration at run time (not at construction time), I believe would be fine to let it optional. This should be fixed whenever we move the decoration to BaseAnnotationLayer instead of AnnotationManager layer

Reply:
>When I see this type Annotation, I understand this represents an instance of an AnnotationTool

This is incorrect. The`Annotation` is _data_ created by an `AnnotationTool`, not an instance of the `AnnotationTool`. `Annotations` are like `toolData` in Cornerstone Legacy.

`isVisible` should be mandatory, and set true by default when the annotation is defined in an `AnnotationTool`'s `addNewAnnotation` method (or as you said before,as a super called to `BaseAnnotationTool`).

There should be no scenario where `isVisible` is undefined, and we should really do the same for `isLocked` too (set it `false` on creation).

cc @sedghi .